pkg/sendconfig/common_workflows.go

```diff
@@ -0,0 +1,49 @@
+package sendconfig
+
+import (
+	"context"
+	"time"
+
+	"github.com/sirupsen/logrus"
+
+	"github.com/kong/kubernetes-ingress-controller/pkg/deckgen"
+	"github.com/kong/kubernetes-ingress-controller/pkg/parser"
+	"github.com/kong/kubernetes-ingress-controller/pkg/store"
+)
+
+// UpdateKongAdminSimple is a helper function for the most common usage of PerformUpdate() with only minimal
+// upfront configuration required. This function is specialized and highly opinionated, other implementations
+// which need more configurability may wish to use the store, parser, deckgen, and PerformUpdate() functionality
```

Comment:
Howso? The limitations of this version/when you'd use it versus the regular functions isn't immediately clear from the function body alone; it'd be useful to summarize that here.

Reply:
This comment has been updated, let me know if the new language reads better! :bow: